### DISCUSSION TITLE
Android开发不使用Unity3D等这类框架，怎么生成Android端开发的SDK

### DISCUSSION THREAD

**Discussion by Junwei9103** [kbengine/kbengine#1481] [Q&A]
移动客户端页面都是用Android普通控件画的，demo里面生成的客户端框架使用Unity等框架（实际开发中不用任何框架），这样我怎么生成移动客户端的SDK？

我现在是这样做的：
1 KBEngine提供的gensdk.bat生成Unity3D版本的客户端SDK插件（C#语言版本）；
2 在Windows系统上用Visual Studio新建Android类库（C#语言），将SDK插件源码导入该新建工程，生成库文件；
3 使用Embeddinator-4000插件将库文件转换成 xxx.aar文件；
4 Android Studio新建工程，将xxx.aar文件作为第三方库导入Android工程去使用。

以上步骤是我的想法，但是我被卡在了第三步，应该是我的一些配置出了问题。
有没有人做过和我类似的工作，有什么经验可以分享下吗？
或者其他的移动端客户端SDK开发相关的经验？

**Comment by Wandar** [comment]
建议按照js插件重写java版的插件,自动生成不用也罢

**Reply by Junwei9103** [reply]
也是一个很好的方案，我去试一下

**Comment by qhf566** [comment]
想找一款成熟的手游产品源码过来我们深度二次开发(全新换套美术，代码也都从新货混淆及优化整一遍)。意向最好是重度的mmo、arpg、角色扮演或动作手游，其他类型的游戏也可以，必须是高品质大厂近5年内做的产品，国内厂商或国外厂商开发的都可以，产品不能在国内运营过的。可以在国外运营过或运行中。准备了足够夸张的米给到你。如朋友那边有可以给推荐一下有米，不墨迹,鹅浩809996677。
